packages/xstate-wallet/src/channel-wallet.ts

```diff
@@ -280,6 +286,14 @@ export class ChannelWallet {
     }
   }
 
+  approveRichObjective(channelId: string) {
```

Comment:
Have you thought up a better term than "Rich objective"?

Reply:
I haven't put much thought into it. Rich objective is a bit of a cop out name though.